src/pe/options.rs

```diff
@@ -16,6 +26,7 @@ impl ParseOptions {
         ParseOptions {
```

Comment:
also i don't know who added this but this does not implement `Default`, but has a method named that instead; this should be fixed to actually implement `Default`

Reply:
thanks for review, just pushed fixes